test/emqx_plugins_SUITE.erl

```diff
@@ -125,9 +128,23 @@ t_ensure_default_loaded_plugins_file(Config) ->
     ok = emqx_plugins:load(),
     {ok, Contents} = file:consult(TmpFilepath),
     DefaultPlugins = default_plugins(),
-    ?assertEqual(DefaultPlugins, lists:sort(Contents)),
+    ?assertEqual(lists:sort(DefaultPlugins), lists:sort(Contents)),
+
+    GenContents = get_loaded_plugins_from_tmpl(),
+    Fun = fun({Name, _}) -> Name end,
```

Comment:
The plugin `enable/disabled` state should be checked too.

Reply:
DONE